src/Microsoft.AspNet.Mvc.Core/LocalRedirectResult.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.Mvc.Logging;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// An <see cref="ActionResult"/> that returns a Redirect to the supplied local Url.
+    /// </summary>
+    public class LocalRedirectResult : ActionResult
+    {
+        private string _url;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalRedirectResult"/> class with the values
+        /// provided.
+        /// </summary>
+        /// <param name="url">The local Url to redirect to.</param>
+        public LocalRedirectResult(string url)
+             : this(url, permanent: false)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalRedirectResult"/> class with the values
+        /// provided.
+        /// </summary>
+        /// <param name="url">The local Url to redirect to.</param>
+        /// <param name="permanent">Specifies if the redirect should be permanent (301) or temporary (302).</param>
```

Comment:
`Specifies if` -> `Specifies whether` 

You need to say which is which also. `if true.... `